src/shared/util/strings.util.ts

```diff
@@ -31,3 +31,8 @@ export const generateRandomString = (
   ]
     .map((b) => CHARSET[b % CHARSET.length])
     .join('')
+
+export function camelToSentence(text) {
+  const result = text.replace(/([A-Z])/g, ' $1')
+  return result.charAt(0).toUpperCase() + result.slice(1)
+}
```

Comment:
Perhaps we can just use this: https://github.com/sindresorhus/decamelize to accomplish this?

And maybe the sister package for the inverse? https://www.npmjs.com/package/camelcase